release/ray_release/byod/build.py

```diff
@@ -124,11 +124,12 @@ def build_anyscale_base_byod_images(tests: List[Test]) -> None:
         and int(time.time()) - start < BASE_IMAGE_WAIT_TIMEOUT
     ):
         for byod_image, test in to_be_built.items():
-            byod_requirements = (
-                f"{REQUIREMENTS_BYOD}_{test.get('python', PYTHON_VERSION)}.txt"
-                if test.get_byod_type() == "cpu"
-                else f"{REQUIREMENTS_ML_BYOD}_{test.get('python', PYTHON_VERSION)}.txt"
-            )
+            py_version = test.get("python", PYTHON_VERSION)
```

Comment:
nit: test.get_python_version()

Reply:
done.. why were you not using that in the past?